app/uk/gov/hmrc/gform/ApplicationLoader.scala

```diff
@@ -182,14 +182,19 @@ class ApplicationModule(context: Context)
   val jsonCrypto =
     SymmetricCryptoFactory.aesCryptoFromConfig(baseConfigKey = "json.encryption", configModule.typesafeConfig)
 
-  val mongoCacheRepository = createMongoCacheRepository("forms", configModule.appConfig.formExpiryDays)
+  val mongoCacheRepository = createMongoCacheRepository("forms")
   val formMongoCache = new FormMongoCache(
     mongoCacheRepository,
     jsonCrypto,
     timeModule.timeProvider
   )
 
-  def createMongoCacheRepository(collectionName: String, expiryDays: Int) = new MongoCacheRepository[String](
+  def createMongoCacheRepository(
+    collectionName: String,
+    expiryDays: Int = configModule.appConfig.formExpiryDays,
+    createdExpiryDays: Int = configModule.appConfig.createdFormExpiryDays,
+    submittedExpiryHours: Int = configModule.appConfig.submittedFormExpiryHours
```

Comment:
Please let's not use default param values.

Reply:
updated